Userland/Applications/FileManager/DirectoryView.cpp

```diff
@@ -114,6 +114,17 @@ void DirectoryView::handle_activation(GUI::ModelIndex const& index)
         return;
     }
 
+    if (path.ends_with(".zip", AK::CaseSensitivity::CaseInsensitive)) {
+        auto result = GUI::MessageBox::show(window(),
+            String::formatted("Extract {} here?", LexicalPath::basename(path)),
+            "Confirm extraction",
+            GUI::MessageBox::Type::Question,
+            GUI::MessageBox::InputType::OKCancel);
```

Comment:
This should be a Yes/No dialog instead of OK/Cancel.

Reply:
Updated to InputType::YesNo.